src/phaser/scene/VillageScene/index.js

```diff
@@ -6,6 +6,7 @@ import Phaser from '@/package/phaser';
 import CharacterFactory from '@/phaser/character/CharacterFactory';
 import CharacterGroup from '@/phaser/character/CharacterGroup';
 import MapManager from '@/phaser/map/MapManager';
+import * as sceneKeys from '@/phaser/scene/sceneKeys';
```

Comment:
```suggestion
import { VILLAGE_SCENE_KEY } from '@/phaser/scene/sceneKeys';
```

전 이 방식으로 사용했었는데 지훈님 방식과 차이가 있네요! 저는 저렇게 쓰니까 `sceneKeys` 처럼 개발자가 임의로 이름을 붙여주지 않아서 코드의 일관성이 더 높아졌다고 생각했는데 대신에 `import`문이 너무 길어지더라구요.. 지훈님처럼 작성했을때 코드의 일관성을 떨어지지만 `import`문이 간결해지고 유지보수가 훨씬 좋아지는 것 같아요. 저는 개인적으로 지훈님 방식이 더 좋네요 😄